Sources/Controllers/CameraViewController.swift

```diff
@@ -173,7 +173,7 @@ public final class CameraViewController: UIViewController {
     NotificationCenter.default.addObserver(
       self,
       selector: #selector(appWillEnterForeground),
-      name: NSNotification.Name.UIApplicationWillEnterForeground,
+      name: NSNotification.Name.NSExtensionHostWillEnterForeground,
```

Comment:
Shouldn't it become `UIApplication.willEnterForegroundNotification`?

Reply:
Your right, updated